src/views/List.jsx

```diff
@@ -1,15 +1,62 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
+import { useEffect } from 'react';
```

Comment:
refactor (nitpick): consolidate the imports

```suggestion
import { useEffect, useState } from 'react';
```

Reply:
Good catch! There's an option here in GitHub to commit your suggestion. Any reasons for or against doing that?

Reply:
Nothing wrong with doing it from GitHub. 😎 The only difference is it will mentioned we co-authored that commit.